.Documentation/articles/toc.yml

```diff
@@ -42,6 +42,10 @@
       href: innoactive-creator/step-inspector.md
     - name: Training Scene Objects
       href: innoactive-creator/training-scene-object.md
+    - name: General Concepts
```

Comment:
Do we have a General Concepts file? I deleted it from my branch again, because we said it should not be part of the conditions documentation ticket.

Reply:
No, it is there because I used your branch as the base, I'll fix it.

Reply:
Fixed, please re-review